modin/experimental/engines/pyarrow_on_ray/frame/partition.py

```diff
@@ -32,24 +56,51 @@ def to_pandas(self):
 
     @classmethod
     def put(cls, obj):
-        """Put an object in the Plasma store and wrap it in this object.
+        """
+        Put an object in the Plasma store and wrap it in this object.
 
-        Args:
-            obj: The object to be put.
+        Parameters
+        ----------
+        obj : object
+            The object to be put.
 
-        Returns:
+        Returns
+        -------
+        PyarrowOnRayFramePartition
             A `RayRemotePartition` object.
         """
         return PyarrowOnRayFramePartition(ray.put(pyarrow.Table.from_pandas(obj)))
 
     @classmethod
     def length_extraction_fn(cls):
+        """
+        Extract the number of rows from the given `pyarrow.Table`.
```

Comment:
this seems wrong, it doesn't extract, it _builds_ the extractor

Reply:
Fixed.